.travis.yml

```diff
@@ -21,12 +21,12 @@ matrix:
     env:
     - CONTAINER_IMAGE=ros:melodic \
       CONTAINER_NAME=industrial_training_ci \
-      SRC_DIRS="exercises/3.0/src exercises/3.1/src exercises/3.2/src exercises/3.3/src"
+      SRC_DIRS="exercises/3.0/ros1/src exercises/3.1/ros1/src exercises/3.2/ros1/src exercises/3.3/ros1/src"
   - name: "Exercise 4"
     env:
     - CONTAINER_IMAGE=ros:melodic \
       CONTAINER_NAME=industrial_training_ci \
-      SRC_DIRS="exercises/4.0/src exercises/4.1/src"
+      SRC_DIRS="exercises/4.0/ros1/src exercises/4.1/ros1/src"
```

Comment:
I think that maintaining both ROS1 and ROS2 in the same branch is getting out of hand - updating these two exercises (mostly) to ROS2 generated 281 file changes.  I'm not sure how many people have strong opinions about this, though.

Reply:
I'm not following why you're suggesting there's a problem. I'm not aware of another option that would be less disruptive.